### DISCUSSION TITLE
How to build a Docker dashboard in Nightingale

### DISCUSSION THREAD

**Discussion by wangbaojin1993** [ccfos/nightingale#1185] [Q&A]
有做好的docker 大盘吗？

**Comment by UlricQin** [comment]
如果有人做过docker大盘，可以把大盘json提交到这里：https://github.com/flashcatcloud/categraf/tree/main/inputs/docker 让社区受益~

**Comment by wangbaojin1993** [comment]
这是来自QQ邮箱的假期自动回复邮件。
 
您好，我最近正在休假中，无法亲自回复您的邮件。我将在假期结束后，尽快给您回复。
